Lib/json/tool.py

```diff
@@ -21,23 +21,42 @@ def main():
                    'to validate and pretty-print JSON objects.')
     parser = argparse.ArgumentParser(prog=prog, description=description)
     parser.add_argument('infile', nargs='?', type=argparse.FileType(),
+                        default=sys.stdin,
                         help='a JSON file to be validated or pretty-printed')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
+                        default=sys.stdout,
                         help='write the output of infile to outfile')
-    parser.add_argument('--sort-keys', action='store_true', default=False,
-                        help='sort the output of dictionaries alphabetically by key')
+    parser.add_argument('--sort-keys', action='store_true',
+                        help='sort the output of dictionaries by key')
```

Comment:
Why removed "alphabetically"?

Reply:
I believe my intention was to make the help message more closely mirror the `json.dump` documentation:

https://github.com/python/cpython/blob/cfa797c0681b7fef47cf93955fd06b54ddd09a7f/Lib/json/__init__.py#L157-L158

Happy to re-add "alphabetically" if that was helpful.